localgov_theme.info.yml

```diff
@@ -24,4 +24,8 @@ regions:
   sidebar_second: "Sidebar second"
   footer_first: "Footer first"
   footer_second: "Footer second"
+  footer_third: "Footer third"
+  lower_footer_first: "Lower footer first"
+  lower_footer_second: "Lower footer second"
+  lower_footer_third: "Lower footer third"
```

Comment:
FAO @Adnan-cds New footer regions declared here.

Reply:
> The footer solution for the localgov_theme is clearly broken.

Hi Andy, I was referring to the current master branch. Sorry, should have been clearer.

Reply:
:-)